data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_feats.lst

```diff
@@ -370,7 +370,7 @@ CATEGORY=FEAT|Toughness.MOD									PRERULE:1,DAMAGE_HP
 CATEGORY=FEAT|Stalwart.MOD													ABILITY:Internal|AUTOMATIC|Stalwart ~ Combat Expertise
 CATEGORY=FEAT|Stalwart.MOD													ABILITY:Internal|AUTOMATIC|Stalwart ~ Fighting Defensively|Stalwart ~ Total Defense Action
 
-Stalwart ~ Combat Expertise			CATEGORY:Internal		VISIBLE:NO																					TEMPBONUS:PC|COMBAT|TOHIT|max(-1*(CombatExpertiseModifier-%CHOICE),(-1*(CombatExpertiseModifier-BAB)))|TYPE=Temporary	TEMPBONUS:PC|VAR|StalwartDRBonus|min(%CHOICE,BAB)	TEMPBONUS:PC|VAR|ClassFeatureDR_ALL|min(StalwartDR,StalwartMaxDRBonus)															TEMPVALUE:MIN=1|MAX=5|TITLE=Combat Expertise trade off
+Stalwart ~ Combat Expertise			CATEGORY:Internal		VISIBLE:NO																					TEMPBONUS:PC|COMBAT|TOHIT|-1*(min(%CHOICE,CombatExpertiseModifier)-CombatExpertiseModifierAttack-CombatExpertiseModifier)|TYPE=Temporary	TEMPBONUS:PC|VAR|StalwartDRBonus|min(%CHOICE,CombatExpertiseModifier)	TEMPBONUS:PC|VAR|ClassFeatureDR_ALL|min(StalwartDR,StalwartMaxDRBonus)															TEMPVALUE:MIN=1|MAX=min(CombatExpertiseModifier,5)|TITLE=Combat Expertise trade off
```

Comment:
Has this been tested? Does it actually work as written?

Reply:
Yes, I've built it locally and use it instead of the latest release.